pkg/webhook/validation.go

```diff
@@ -133,6 +134,10 @@ func (v *validator) validateBundle(ctx context.Context, bundle *trustapi.Bundle)
 				unionCount++
 			}
 
+			if source.UseDefaultCAs != nil && *source.UseDefaultCAs {
+				unionCount++
+			}
+
 			if unionCount != 1 {
 				el = append(el, field.Forbidden(
 					path, fmt.Sprintf("must define exactly one source type for each item but found %d defined types", unionCount),
```

Comment:
Does this line mean that we cannot combine eg. Secret + Defaults?

Reply:
It does in one entry - but you can have multiple entries, and they're all combined at the end.

So you can't have:

```text
- configMap: ...
  useDefaultCAs: true
```

but you can have:

```text
- configMap: ...
- useDefaultCAs: true
```

Reply:
ah makes sense